src/docs/product/cli/dif.mdx

```diff
@@ -55,32 +55,37 @@ folder or ZIP archive.
 
 ## Creating Source Bundles
 
-To get inline source context in stack traces in the Sentry UI, `sentry-cli` can
-scan debug files for references to source code files, resolve them in the local
-file system and bundle them up. The resulting source bundle is an archive
-containing all source files referenced by a specific debug information file.
+To get inline source context in stack traces in the Sentry UI, Sentry needs so-called
+__source bundles__, an archive containing all source files referenced by a specific
+debug information file. You can use sentry-cli to scan debug files for references to
+source code files, resolve them in the local file system and bundle them up in a source
+bundle. Either use the `difutil bundle-sources` command or add the `--include-sources`
+option to the `upload-dif` command.
```

Comment:
```suggestion
To get inline source context in stack traces in the Sentry UI, Sentry needs
__source bundles__, an archive containing all source files referenced by a specific
debug information file. You can use sentry-cli to scan debug files for references to
source code files, resolve them in the local file system, and bundle them up in a source
bundle. 

This is particularly useful when building and uploading debug information files
are detached, because a source bundle can be created when building, and then uploaded with `sentry-cli upload-dif` at any time later.

```

To me, the original way this was written is easier to read. It's confusing to give the instructions "Either use the `difutil bundle-sources` command or add the `--include-sources`
option to the `upload-dif` command." and then show these commands again below. It makes more sense to talk about the commands and then show them immediately afterwards. I also think it makes more sense to mention that this method is particularly useful when building and uploading debug information files are detached, the way we have it in the original.